### PR TITLE
add tutanota.toml corrections

### PR DESCRIPTION
**Pull request by ganthern**
**Type of pull request:** product edit

**Related issues:** N/A

---

There's a few inaccuracies and/or misleading statements in the Tutanota survey.

### CODE REVIEW

**Comment by milesmcc** [comment]
So this brings up an interesting case: What do we do when a company is legally required to follow certain rules based on their jurisdiction, but do not explicitly say so in their privacy policy? We've dealt with this question before, but it's worth reconsidering. @doamatto and @ibarakaiev — what do you all think?

File: products/tutanota.toml [hunk 1]
```diff
@@ -30,9 +30,9 @@ citations = [
 ]
 
 [rubric.data-breaches]
-value = "no"
+value = "yes-72"
 notes = [
-  "Although their imprint describes them being based in Germany, Tutanota does not comply with Article 33 of the GDPR."
+  "Tutanota is based in Germany so it is legally obliged to notify users of data breaches, even if not listed in the privacy policy."
```
Comment (milesmcc) [inline products/tutanota.toml:35]: This is not quite right — our interpretation of GDPR article 33 is that the company must notify the _supervisory authority_, not users themselves:

> In the case of a personal data breach, the controller shall without undue delay and, where feasible, not later than 72 hours after having become aware of it, notify the personal data breach to the supervisory authority competent in accordance with [Article 55](https://gdpr-info.eu/art-55-gdpr/), unless the personal data breach is unlikely to result in a risk to the rights and freedoms of natural persons.

**Comment by doamatto** [comment]
There are really two ways to think of it: we can assume they’d play by the rules that they will notify of breaches, even if they don’t explicitly say they will, or we can have their word, per se, that they will.

I can’t say that I’m in favour of the optimistic or pessimistic view, but I do think it should be explicitly mentioned that they follow the policy.

File: products/tutanota.toml (hunk 1 above)
Comment (doamatto) [reply products/tutanota.toml:35]: Supervisory authority is defined in Art 4, §22

> supervisory authority’ means an independent public authority which is established by a Member State pursuant to [Article 51](https://gdpr-info.eu/art-51-gdpr/)

**Pull request merged**